
syntax = "proto3";

// The greeting service definition.
service Greeter {

  // Uploads a file
  rpc UploadFile(FileMessage) returns (StringResponse) {}
  // Downloads a file
  rpc DownloadFile(StringResponse) returns (FileMessage) {}
}


message StringResponse {
  string message = 1;
}

message FileMessage {
  bytes chunk_data = 1;
}
